libcnb/src/build.rs

```diff
@@ -190,4 +197,22 @@ impl BuildResultBuilder {
         self.store = Some(store);
         self
     }
+
+    /// Adds a build SBOM to the build result.
+    ///
+    /// Entries in this SBOM represent materials in the build container(sic!) for auditing purposes.
```

Comment:
I think we should remove the `(sic!)` and use a corrected description of what the SBOM represents, rather than use the wording from upstream verbatim.

I also found this alternative description upstream, which doesn't have that mistake (no need to use this verbatim either; just mentioning for inspiration):

> Write a partial Software Bill of Materials to <layers>/build.sbom.<ext> describing any provided build dependencies not associated with a layer.

https://github.com/buildpacks/spec/blob/main/buildpack.md#purpose-2

Reply:
Oh, this is not a mistake. At least I didn't understand it as such. From my understanding, the `build` SBOM is strictly for things that got added to the build _container_ (hence the sic! as it might be unexpected). The SBOMs added for build aren't in any way connected to the resulting image.

The description you linked isn't wrong, but is omitting the fact that that BOM only describes the build container and that it won't be associated with the launch image at all.

Reply:
I think the use of "sic" here makes the meaning of this more confusing, not less. Could you reword without it?

Reply:
I think in general we might need to give some examples of the types of things that belong in each category of SBOM, given buildpack authors might not know otherwise. Though I know this is partly subjective, and maybe something we'll just have to figure out over time :-)

Reply:
Fixed (I think?) in [1f99649](https://github.com/heroku/libcnb.rs/pull/489/commits/1f99649540732d2f07d0275f782d3b7e01949157). Agreed that it might be confusing - you did comment on it for a reason. ;-)